lib/data_update_scripts/20200709154442_update_broadcastable_for_broadcasts.rb

```diff
@@ -0,0 +1,16 @@
+module DataUpdateScripts
+  class UpdateBroadcastableForBroadcasts
+    def run
+      Broadcast.find_each do |cast|
+        case cast.broadcastable_type
+        when "Welcome"
+          broadcastable = WelcomeNotification.create
+          cast.update!(broadcastable: broadcastable)
+        when "Announcement"
+          broadcastable = Announcement.create(banner_style: cast.banner_style)
+          cast.update!(broadcastable: broadcastable)
```

Comment:
One suggestion here, rather than forcing this entire job to fail if you run into one back record a more resilient solution might be to report the failure to Honeybadger and continue to update the rest of the records so you can at least get as many done as possible. Was thinking something like this
```ruby
result = cast.update(broadcastable: broadcastable)
next if result
Honeybadger.context(broadcast_id: cast.id, errors: cast.errors_as_sentence)
Honeybadger.nofity("Broadcast Update Failed")
```

Reply:
Oh, I like this a lot! Making the change now. 😄

Reply:
I've updated this to (hopefully) reflect the error logging you suggested - let me know what you think @mstruve!